FirebaseSessions.podspec

```diff
@@ -35,12 +35,15 @@ Pod::Spec.new do |s|
 
   base_dir = "FirebaseSessions/"
   s.source_files = [
-    base_dir + 'Sources/**/*.swift',
+    base_dir + 'Sources/**/*.{swift,h,m}',
+    base_dir + 'Protogen/**/*.{c,h,m,mm}',
```

Comment:
Heads up that this will unfortunately not be as straightforward when setting up SwiftPM support due to the no mixed languages constraint. There are some workarounds, however.

And also, we should be able to call C from Swift... that might be able to help us.

Reply:
Ooh that's a really good callout. Do you know of these workarounds? I would love to learn this power. Also, this code could easily be written in C if that works (in fact I believe it is valid C).

Reply:
> Do you know of these workarounds? I would love to learn this power. 

I could be way off but I'm getting prequels Palpatine x Anakin vibes 😄 

But yes! We could create a target called something like `SessionsProtoLib` that the main Sessions SwiftPM target depends on. That should work. So basically just separate the Sessions SDK into a Swift target that depends on a C based target with the proto stuff. 

An alternative idea that should avoid all of this (and is probably easier) is to use: https://github.com/apple/swift-protobuf which is Apple's official Swift proto library.

Reply:
I would like the idea of using SwiftProtobuf for Swift libraries, but SwiftProtobuf doesn't support binary distributions. See https://github.com/firebase/firebase-ios-sdk/issues/8200 and https://github.com/apple/swift-protobuf/issues/1101

Reply:
@samedson, looked into and we are going to continue to use nanopb. LMK if you need help structuring the SPM stuff